app/controllers/users_controller.rb

```diff
@@ -64,4 +67,20 @@ def should_display_user?(user)
   def get_user
     @user = User.friendly.find(params[:id])
   end
+
+  def count_timezone_degrees
+    unless params[:timezone_select].blank?
+      if @current_user && @current_user.longitude
+        lng = @current_user.longitude
```

Comment:
if it is possible that `@current_user` maybe nil, then `lng = @current_user.try(:longitude)`